docs/en_US/NNICTLDOC.md

```diff
@@ -650,3 +651,37 @@ Debug mode will disable version check function in Trialkeeper.
     ```bash
     nnictl --version
     ```
+
+<a name="hdfs"></a>
+
+![](https://placehold.it/15/1589F0/000000?text=+) `Manage hdfs`
+
+* __nnictl hdfs set__
+  * Description
+
+    set the host and userName of hdfs
+
+  * Usage
+
+    ```bash
+    nnictl hdfs set [OPTIONS]
+    ```
+
+  * Options
+
+  |Name, shorthand|Required|Default|Description|
+  |------|------|------ |------|
+  |--host|  True| |The host of hdfs|
+  |--user_name|  True| |The userName of hdfs|
+
+* __nnictl hdfs clean__
+
+  * Description
+
+    Clean up the files created by nni in hdfs
```

Comment:
what files? a specific experiment, or all finished experiments?

Reply:
NNI will upload code file to hdfs, and PAI will use these files to start container. These files is hidden for users, and will not be used again after the experiment is started. NNI should clean up all of hidden files.